src/bson.rs

```diff
@@ -397,19 +397,19 @@ impl Bson {
         match *self {
             Bson::RegExp(ref pat, ref opt) => {
                 doc! {
-                    "$regex" => (pat.clone()),
```

Comment:
I'm so confused about git saying the first ')' was removed here but the second below